backend/location/models.py

```diff
@@ -10,6 +10,22 @@ class Location(models.Model):
         max_length=4096,
     )
 
+    place_name = models.CharField(
+        help_text="If it is a specific place (e.g. city hall) this contains the name of the place",
+        verbose_name="Place Name",
+        max_length=1024,
+        blank=True,
+        null=True,
+    )
+
+    exact_address = models.CharField(
+        help_text="Points to the exact address the location is on (e.g. 'Silk Road 50')",
+        verbose_name="Exact Address",
+        max_length=1024,
+        blank=True,
+        null=True,
+    )
```

Comment:
This does introduce street address. Tbh I thought the fields exact_address and place_name were sufficient looking at many examples of address data from Nominatim. Since we store place_id and osm_id we can always catch the full data from Nominatim again, I don't think we need more in our database than we use.
What would be the advantage of storing exact locations in a different table? They have the same data structure and seperate place_id's and osm_id's and geodata so they are always unique. They just additionally fill one or both of the newly added fields. Adding different tables would make location such much more complicated because of course if somebody types in "San Francisco" you also want to show events taking place in "San Francisco city hall". But if you had those locations in two different tables you'd have to search both tables to find all relevant locations.

Reply:
Okay, then it makes sense to keep in the same table.